src/app.js

```diff
@@ -1,115 +1,78 @@
-var SVG_NS = "http://www.w3.org/2000/svg"
-
 export default function (app) {
-  var view = app.view || function () {
-    return ""
-  }
-
-  var model
+  var state = {}
```

Comment:
In this case it's fine because the first plugin from which we'll try merge the local state variable with is actually the options object passed into app. We'll have an annotated hyperapp someday which is better than comments!